packages/worker/src/tests/api/users.ts

```diff
@@ -91,11 +91,11 @@ export class UserAPI {
 
   // USER
 
-  saveUser = (user: User, status?: number) => {
+  saveUser = (user: User, status?: number, headers?: any) => {
```

Comment:
I'd like to start promoting this pattern for the optional args of status and headers:
https://github.com/Budibase/budibase/blob/develop/packages/worker/src/tests/api/users.ts#L135-L140

Reply:
Not going to implement that at the moment as I'm working off master branch, and that pattern is currently on develop.